src/main/java/frc/lib/lib2706/SubsystemChecker.java

```diff
@@ -27,15 +27,17 @@ public enum SubsystemType {
         VisionNTSubsystem,
         BlingSubsystem,
         GripperSubsystem,
+        ElevatorSubsystem
     };
 
     /**
      * Allowed Subsystems for each robot
      */
-    // RobotID: 0, 2023 Competition robot, unnamed
+    // RobotID: 0, 2025 Competition robot, unnamed
     private static SubsystemType[] compBotId0 = new SubsystemType[] {
         SubsystemType.SwerveSubsystem,
         SubsystemType.ArmSubsystem,
+        SubsystemType.ElevatorSubsystem,
```

Comment:
comment out this line for now, until we have the competition robot. Otherwise, Apollo won't work.
Add ElevatorSubsystem in beetleId2